gradle/check-licenses.gradle

```diff
@@ -151,6 +151,7 @@ downloadLicenses {
     // RocksDB is dual licensed under Apache v2.0 or GPL 2 licenses
     // Explicitly declare that we are using the Apache v2.0 license
     (group('org.rocksdb')) : apache,
+    (group('org.apache.tuweni')) : apache,
```

Comment:
why did we need this?

Reply:
Think I brought that from another branch I was testing. Have removed it.